scripts/main.js

```diff
@@ -18,22 +18,26 @@ for(const key in workgroups) {
 	const doc = Templates.workgroup(wg);
 	writeFileSync(joinPath(docsPath, `${wg.key}.md`), doc);
```

Comment:
Are you sure this path exists?

Reply:
Actually above we just cleared it with rimraf. I can do a mkdirp if you like :]